src/styles/blocks/head-block/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  padding: 0 50px;
+  background-color: $background_white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
The color value in the box-shadow property could use a variable for better maintainability and consistency across the project.